fv_be/app/models/sites.py

```diff
@@ -0,0 +1,133 @@
+from django.core.validators import validate_slug
+from django.db import models
+
+from fv_be.app.models import BaseModel
+from fv_be.users.models import User
+
+from .constants import Role, Visibility
+
+
+class LanguageFamily(BaseModel):
```

Comment:
@caforbes Is this a good name for this model? It is pulling from the "FVLanguage" documents in nuxeo, but thought we might want to keep "Language" available as a name for future use. This is the model for the group headings that are shown on the Explore Languages page.

Reply:
No, this is very confusing to me because "language family" is a different level of grouping which I would expect to map to FVLanguageFamily. (See the FPCC language status report which is grouped by family < language).

I would stick with Language here and replace all references to "language family" with just "language". I don't think anything would replace this core concept. (Future-feature-wise, sites with multiple "languages" will likely not be cross-"Language" in this sense, but cross-community or cross-Site).

We could discuss storing the language family information too, since this is known information. This is what the nuxeo directory structure was meant to encode -- it would be better in a 1:M relation with Language. However, I don't think we currently leverage this info in any feature, so I wouldn't prioritize it.